tests/test_vm_mount.cpp

```diff
@@ -20,44 +20,123 @@
 #include <multipass/vm_mount.h>
 
 namespace mp = multipass;
+namespace mpt = multipass::test;
 using namespace testing;
 
 namespace
 {
 struct TestVMMount : public Test
 {
-    mp::VMMount a_mount{"asdf", {{1, 2}, {2, 4}}, {{8, 4}, {6, 3}}, multipass::VMMount::MountType::Classic};
+    static const mp::VMMount a_mount;
 };
 
+struct TestUnequalVMMountsTestSuite : public Test, public WithParamInterface<std::tuple<mp::VMMount, mp::VMMount>>
+{
+};
+
+const mp::VMMount TestVMMount::a_mount{"asdf", {{1, 2}, {2, 4}}, {{8, 4}, {6, 3}}, mp::VMMount::MountType::Classic};
+
+TEST_P(TestUnequalVMMountsTestSuite, CompareMountsUnequal)
+{
+    auto [mount_a, mount_b] = GetParam();
+
+    // Force use of overloaded operator== and operator!=
+    EXPECT_TRUE(mount_a != mount_b);
+    EXPECT_FALSE(mount_a == mount_b);
```

Comment:
This is to force the use of the `operator==()` and the `operator!=()` functions and cover them both. Because `a_mount` and `b_mount` are supposed to be unequal, there's no way to use `EXPECT_EQ` and force the use of `operator==()`.

Reply:
Ah, I understand. You could always do something like `EXPECT_EQ(mount_a, mount_a)`, since that's pretty much an essential test for `operator==()` to pass. But I think this is a trivial thing, so we'll keep it as it is.